Salesforce_Advanced/Create_Record_and_File/CreateRecordAndFile.xml

```diff
@@ -0,0 +1,313 @@
+										<update valuesource="fieldvalue" field="unsavedCase" enclosevalueinquotes="false" value="false"/>
+									</updates>
+								</action>
+								<action type="unblockUI"/>
+							</actions>
+						</skuid__button>
+						<skuid__button uniqueid="sk-16eI-82214" label="Cancel">
+							<actions>
+								<action type="cancel">
+									<models>
+										<model>case</model>
+									</models>
+								</action>
+								<action type="action-sequence" action-sequence-id="ccaddeb8-d57b-4bb9-93bb-a7cabcb8a3f3">
+									<onerroractions>
+										<action type="blockUI" message="There was an error" timeout="3000"/>
+									</onerroractions>
+								</action>
+							</actions>
+						</skuid__button>
+					</buttons>
+				</skuid__buttonGroup>
+			</groups>
+		<styles>
+				<spacing bottom="4" left="4" right="4"/>
+			</styles>
+		</skuid__buttonSet>
+	</components>
+</division>
+</divisions>
+</skuid__grid>
+</components>
+<styles>
+	<spacing top="4" bottom="4" left="4" right="4"/>
+</styles>
+<background type="color" color="white"/>
+</skuid__wrapper>
+</components>
+</division>
+<division alignSelf="auto" minWidth="100px" ratio="1">
+	<components>
+		<skuid__wrapper uniqueid="sk-16ev-11970">
+			<components>
+				<skuid__wrapper uniqueid="sk-16ZF-36917">
+					<components>
+						<skuid__table allowColumnFreezing="dragDrop" model="contentDocument" uniqueid="sk-16cm-77455" mode="read" showSaveCancel="false" title="Content Document" pageSize="10" showListFooter="false">
+							<fields>
+								<field id="CreatedById" uniqueid="fi-16cm-78074"/>
+								<field id="CreatedBy.Name" uniqueid="fi-16cm-78075"/>
+								<field id="CreatedDate" uniqueid="fi-16cm-78076"/>
+							</fields>
+							<filtering enableSearch="false"/>
+							<rowActions/>
+							<massActions/>
+						</skuid__table>
+						<skuid__table allowColumnFreezing="dragDrop" model="contentDocumentLink" uniqueid="sk-16eV-87109" mode="read" showSaveCancel="false" title="Document Link" pageSize="10" showListFooter="false">
+							<fields>
+								<field id="ContentDocument.Title" uniqueid="fi-16eV-87707"/>
+								<field id="ContentDocument.CreatedById" uniqueid="fi-16eV-87709"/>
+								<field id="LinkedEntityId" uniqueid="fi-16eV-87711"/>
+							</fields>
+							<filtering enableSearch="false"/>
+							<rowActions/>
+							<massActions/>
+							<styles>
+								<spacing top="4"/>
+							</styles>
+						</skuid__table>
+					</components>
+					<styles>
+						<spacing top="4" bottom="4" left="4" right="4"/>
+					</styles>
+					<background type="color" color="white"/>
+				</skuid__wrapper>
+			</components>
+			<styles>
+				<spacing top="4" bottom="4" left="4" right="4"/>
+			</styles>
+			<background type="color" color="white"/>
+		</skuid__wrapper>
+	</components>
+</division>
+</divisions>
+</skuid__grid>
+	</components>
+	<resources>
+		<javascript>
+			<jsitem location="inline" name="newInlineJS" cachelocation="false" url="">(function(skuid){
```

Comment:
```suggestion
			<jsitem location="inline" name="unsavedRecordCheck" cachelocation="false" url="">(function(skuid){
```

Reply:
This snippet's purpose is explained in the readme but may as well give it an informative name.